health-services/project-factory/src/server/utils/campaignFacilitiesUtils.ts

```
@@ -0,0 +1,674 @@
+        );
+    }
+    logger.info("Updating active facilities to inactive completed for current chunk");
+}
+
+async function persistNewInactiveFacilities(allFacilityList: any[], campaignFacilities: any[], campaignNumber: string, userUuid: string) {
+    logger.info("Persisting new inactive facilities for current chunk");
+    const setOfFacilityNamesFromDBForCurrentCamoaign = new Set(
+        campaignFacilities?.map((facility: any) => facility.name)
+    )
+    const newInactiveFacilitiesToBePersisted = []
+    for(let facility of allFacilityList) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'facility' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)